### ISSUE TITLE
[Feature] Customizable format

### ISSUE THREAD

**Issue by beeb** [sdnts/lumbermill-rs#5]
As discussed a few weeks/months ago, having a fully customizable format would be awesome!

We could provide a few presets that would match the current existing ones.

I would have the following parameters, at least in a first step:

- datetime format (date + time or only time)
- level format (full or 3-letter)
- keys format (compact -- timestamp, level and message are not shown, or all keys are shown)
- color (full color, dimmed timestamp + mod + src, or monochrome)

**Comment by sdnts** [comment]
Right, I remember! I've finally got some time to look into this soon.
